src/components/project/list.vue

```diff
@@ -50,24 +56,71 @@ except according to the terms contained in the LICENSE file.
 import Loading from '../loading.vue';
 import PageSection from '../page/section.vue';
 import ProjectNew from './new.vue';
-import ProjectRow from './row.vue';
+import ProjectHomeBlock from './home-block.vue';
+import ProjectSort from './sort.vue';
 
 import modal from '../../mixins/modal';
 import routes from '../../mixins/routes';
 import { requestData } from '../../store/modules/request';
 
 export default {
   name: 'ProjectList',
-  components: { Loading, PageSection, ProjectNew, ProjectRow },
+  components: {
+    Loading,
+    PageSection,
+    ProjectNew,
+    ProjectHomeBlock,
+    ProjectSort
+  },
   mixins: [modal(), routes()],
   data() {
     return {
       newProject: {
         state: false
-      }
+      },
+      sortMode: 'latest'
```

Comment:
I think at one point we talked about making alphabetical the default so that the default matches what's shown today. (Just randomly noticed that while taking a look at the QA server.)

Reply:
After the tuesday meeting, we're going to try leaving it as `latest` and get feedback from issa.